appserver/security/jacc.provider.file/pom.xml

```diff
@@ -80,30 +78,32 @@
 
     <dependencies>
         <dependency>
-            <groupId>javax.security.jacc</groupId>
-            <artifactId>javax.security.jacc-api</artifactId>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
         </dependency>
         <dependency>
-             <groupId>javax.servlet</groupId>
-             <artifactId>javax.servlet-api</artifactId>
+            <groupId>javax.security.jacc</groupId>
+            <artifactId>javax.security.jacc-api</artifactId>
```

Comment:
Is there a reason for rearranging the order of dependencies?

Reply:
Yes absolutely. The jakarta servlet dependency should take precedence. A couple of other dependencies still refer to the older javax dependency, and in their current order they would have precedence.